server/pulp/server/webservices/controllers/repositories.py

```diff
@@ -362,8 +363,21 @@ def GET(self, repo_id, importer_id):
         importer = importer_manager.get_importer(repo_id)
         return self.ok(importer)
 
-    @auth_required(UPDATE)
+    @auth_required(DELETE)
     def DELETE(self, repo_id, importer_id):
+        """
+        Remove an importer from a repository.
+
+        :param repo_id:     The id of the repository to remove the importer from
+        :type  repo_id:     str
+        :param importer_id: The id of the importer to remove from the given repository
+        :type  importer_id: str
+        """
+        # Raise a MissingResource exception if the repo or the importer doesn't exist
+        importer_manager = manager_factory.repo_importer_manager()
+        importer = importer_manager.get_importer(repo_id)
+        if importer['id'] != importer_id:
+            raise exceptions.MissingResource(importer_id=importer_id)
```

Comment:
@beav Although this isn't a subclass of PulpCodedException, @barnabycourt did add the 'error' field with an error code, PLP0009, so as a user will see something like 
`{"http_request_method": "GET", ...., "error": {"code": "PLP0009", "data": {"resources": {"resource_id": "notarepo"}}}`